src/components/Form/FieldPassword/index.tsx

```diff
@@ -56,7 +58,11 @@ export const FieldPassword = <
               <IconButton
                 isDisabled={props.isDisabled}
                 onClick={() => setShowPassword((x) => !x)}
-                aria-label={showPassword ? 'Hide password' : 'Show password'} // TODO: translation
+                aria-label={
+                  showPassword
+                    ? t('components:fieldPassword.passwordHide')
+                    : t('components:fieldPassword.passwordShow')
```

Comment:
```suggestion
                    ? t('components:fieldPassword.hidePassword')
                    : t('components:fieldPassword.showPassword')
```
je trouve ca plus logique dans ce sens